app/assets/stylesheets/config/_base.css

```diff
@@ -28,6 +29,10 @@
   border: 0.5px solid blue;
 }
 
+html, body {
+  height: 100%;
+}
```

Comment:
**question:** why is this needed?

Reply:
I used this logic for the footer to stay in the bottom:
```
<body>
  <div class="content">
    content
  </div>
  <footer class="footer"></footer>
</body>

html, body {
  height: 100%;
}
body {
  display: flex;
  flex-direction: column;
}
.content {
  flex: 1 0 auto;
}
.footer {
  flex-shrink: 0;
}
```

Without the set height it doesn't work. I admit though that I don't quite understand why it doesn't work without it. I need to read a bit more on it.